front-end/src/components/ui/Loader/Loader.css

```
@@ -1,13 +1,13 @@
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  z-index: 1000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
 
-    backdrop-filter: blur(5px);
+  backdrop-filter: blur(3px);
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)